process-automation/latest/model/connectors/aws.md

```diff
@@ -730,6 +567,49 @@ The output parameters of the Transcribe connector are:
 | transcription | String | *Required.* Transcription result. |
 | webVTT | JSON | *Optional* Subtitles result in Web Video Text Tracks format. |
 
+### Transcribe configuration parameters
+
+The configuration parameters for the Transcribe connector are:
+
+| Parameter | Description |
+|-----------|-------------|
+| AWS_ACCESS_KEY_ID | *Required.* The access key to authenticate against AWS. |
+| AWS_SECRET_KEY | *Required.* The secret key to authenticate against AWS. |
+| AWS_REGION | *Required.* The region of AWS to use the Textract service in. |
+| AWS_S3_BUCKET | *Required.* The name of the S3 bucket to use. |
+| AWS_TRANSCRIBE_LANGUAGES | List of comma separated languages that are spoken in the audio/video file. |
+
+### Transcribe errors
+
+The possible [errors]({% link process-automation/latest/model/connectors/index.md %}#errors) that can be handled by the Transcribe connector are:
+
+| Error | Description  |
+|-------|--------------|
+| MISSING_INPUT | A mandatory input variable was not provided. |
+| INVALID_INPUT | The input variable has an invalid type. |
+| INVALID_RESULT_FORMAT | The REST service result payload cannot be parsed. |
+| LIMIT_EXCEEDED | The service limit was exceeded. |
+| ACCESS_DENIED | The user is not authorized to perform the action. |
+| INTERNAL_FAILURE | An internal Amazon Lex error occurred. |
+| UNKNOWN_ERROR | Unexpected runtime error. |
+| BAD_REQUEST | The server could not understand the request due to invalid syntax. |
+| UNAUTHORIZED | The request has not been applied because it lacks valid authentication. |
+| FORBIDDEN | The server understood the request but refused to authorize it. |
+| NOT_FOUND | The server could not find what was requested. |
+| METHOD_NOT_ALLOWED | The request method is known by the server but is not supported. |
+| NOT_ACCEPTABLE | The server cannot produce a response matching the list of acceptable values. |
+| REQUEST_TIMEOUT | The server would like to shut down this unused connection. |
```

Comment:
I would write "is requesting" instead of "would like" because the server doesn't have feelings :)

Reply:
yes